gojira.sh

```diff
@@ -415,7 +415,7 @@ Options:
   -r,  --repo           repo to clone kong from
   -pp, --port           expose a port for a kong container
   -v,  --volume         add a volume to kong container
-  -e,  --env            add environment variable to kong container
+  -e,  --env            add environment variable binding to kong container in the form: var=value
```

Comment:
maybe it's just me, but this line feels too long, something like

```
-e,  --env KEY=VAL           add environment variable binding to kong container
```

Reply:
@eskerda agreed. Changed to match your suggestion in https://github.com/Kong/gojira/pull/10/commits/7072eb92c330db09aacf286413b00311f16d3fb9